### ISSUE TITLE
Empty translate page

### ISSUE THREAD

**Issue by zhum** [fastaddons/WebTranslate#13]
**Describe the bug**  
The addon worked fine for some time, but eventually I started get just empty window with only menu on the left. I tried to change translators (google, ms, yandex), but this didn't help. I suspect it can be interference with other addons or something else. I would like to help to debug it (but I don't know now how).

**Steps To Reproduce**  
1. Open Thunderbird, open any email.
2. Select any word (e.g. "Hello"), right click on it, select "translate 'Hello'".
3. New popup window appears, but no translation 
![webtranslate-error](https://user-images.githubusercontent.com/336958/215731313-3cbabff5-310b-4732-9ed3-8efb27c8e09f.png)

**Expected behavior**  
New popup window with the translator page appears.

**Device info:**
 - operating system: Ubuntu 22.04.1 LTS
 - Browser and version: Thunderbird 102.7.0 (64-bit)
 - addon version: Web Translate 4.3

**Comment by fastaddons** [comment]
Hello,
Sometimes somehow it's just takes a few seconds to load. I'm using DeepL and when I tested it now, it took more than 10 seconds to load it. But reopening it again and again was then fast again.
But I've tested other translator engines and they seem to work fast.

Try to restart Thunderbird, that could help as well.
If it doesn't help, try to uninstall my addon, then restart Thunderbird and then install it again. That will clear all caches and databases so that should fix whatever is broken.
